### PR TITLE
Detail page fixtures part 1

### PR DESCRIPTION
**Pull request by chrisvanmook**
I've added a lot of fixtures for detail pages (more to come in second PR, otherwise this would be an even bigger PR)
Aksi removed unused redux action + removed possible failure in getDetailPageData method when URL parameters are used

### CODE REVIEW

File: src/api/panorama/thumbnail/getPanoramaThumbnail.ts [hunk 1]
```diff
@@ -92,7 +92,9 @@ export async function getPanoramaThumbnail(
   return transformResponse(response)
 }
 
-function transformResponse(response: RawResponse): PanoramaThumbnail | null {
+function transformResponse(
+  response: RawResponse,
+): (Omit<PanoramaThumbnail, 'pano_id'> & { id: string }) | null {
```
Comment (jonkoops) [inline src/api/panorama/thumbnail/getPanoramaThumbnail.ts:97]: Perhaps it's time we refactor the code around this method to accept a plain `PanoramaThumbnail` without these weird changes.

File: src/app/components/PanoramaViewer/PanoramaViewer.test.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { fireEvent, render } from '@testing-library/react'
 import PanoramaViewer from './PanoramaViewer'
 
 const mockPush = jest.fn()
+jest.mock('marzipano')
```
Comment (jonkoops) [inline src/app/components/PanoramaViewer/PanoramaViewer.test.tsx:5]: You might want to pull in the latest changes from `develop`. I've updated this in another PR that has landed.

File: src/app/utils/useMarzipano.test.ts [hunk 3]
```diff
@@ -2,6 +2,8 @@ import { renderHook } from '@testing-library/react-hooks'
 
 import useMarzipano from './useMarzipano'
 
+jest.mock('marzipano')
```
Comment (jonkoops) [inline src/app/utils/useMarzipano.test.ts:5]: Same here, this has been fixed upstream already.

File: src/store/redux-first-router/actions.js [hunk 4]
```diff
@@ -181,22 +182,6 @@ export const toConstructionFilesFromEndpoint = (endpoint) => {
 export const toDatasetSearch = toSearchOfType(routing.datasetSearch.type)
 export const toSearch = toSearchOfType(routing.search.type)
 
-export const toDataSuggestion = (payload, view) => {
```
Comment (jonkoops) [inline src/store/redux-first-router/actions.js:184]: No longer used? Nice.

**Review by jonkoops — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Review by janjaap — APPROVED** [review]

**Pull request merged**